### PR TITLE
change link to godoc from v1 to v5

### PR DESCRIPTION
**Pull request by toshitanian**
### Proposed Changes

change link to godoc from v1 to v5

#### Acceptance Test Output

Just updates one line in  README

### CODE REVIEW

File: README.md
Comment (paddycarey) [inline README.md:10]: Should we perhaps update the link to point at https://pkg.go.dev/gopkg.in/auth0.v5 ? godoc.org now redirects there after landing anyway, we could save a hop.

**Comment by toshitanian** [comment]
godoc.org did not redirect me to v5 doc. And the v1 doc has deprecated methods of this library.
For instanfce,
`func NewConnectionManager(m *Management) *ConnectionManager`
I got stuck by trying to use `NewConnectionManager` in order to create a ConnectionManager which is not supported in `v5` version.
We should update the link for the future users.

Untitled

**Comment by paddycarey** [comment]
Sorry, what I mean is should we _also_ update the link to point at `pkg.go.dev` in _addition_ to the version update?

**Comment by toshitanian** [comment]
ah. understood.  The domain has been changed. I will take a look and update this pull request.
Thanks a lot!

**Comment by toshitanian** [comment]
@paddycarey 
I made change at the commit of https://github.com/go-auth0/auth0/pull/190/commits/5aa46f579c8b466b9f8acad15527a57088227582. Does this make sense?


Thank you so much again for helping me with my update.

**Review by paddycarey — APPROVED** [review]
LGTM

**Comment by toshitanian** [comment]
Thanks! It seems that I'm not allowed to merge in this repo.
I appreciate if you could merge this 😄

**Comment by paddycarey** [comment]
I believe we'll need @alexkappa to hit the button for us, thanks for your contribution!

**Comment by toshitanian** [comment]
Updated status badge from godoc.org to pkg.go.dev as well.

**Comment by toshitanian** [comment]
@alexkappa 

Hi, could you merge this if it looks good?

**Pull request merged**
